### ISSUE TITLE
运行helixfold_single_inference.py  报错

### ISSUE THREAD

**Issue by xiong3134** [PaddlePaddle/PaddleHelix#205]
AttributeError: module 'paddle.fluid.core_avx.eager.ops' has no attribute 'fused_gate_attention

**Comment by LihangLiu** [comment]
Hi, which version of PaddlePaddle are you using? To run HelixFold-Single, you need to install PaddlePaddle dev as stated [here](https://github.com/PaddlePaddle/PaddleHelix/tree/dev/apps/protein_folding/helixfold-single#installation).

**Comment by xiong3134** [comment]
i  have  installed dev 0.0.0. CPU version ,but is also "AttributeError: module 'paddle.fluid.core_avx.eager.ops' has no attribute 'fused_gate_attention",  it  must  install  0.0.0 GPU  version?

**Comment by xiong3134** [comment]
> i have installed dev 0.0.0. CPU version ,but is also "AttributeError: module 'paddle.fluid.core_avx.eager.ops' has no attribute 'fused_gate_attention", it must install 0.0.0 GPU version?

i have installed dev 0.0.0. CPU version ,but is also "AttributeError: module 'paddle.fluid.core_avx.eager.ops' has no attribute 'fused_gate_attention", it must install 0.0.0 GPU version?

**Comment by LihangLiu** [comment]
Yes, `fused_gate_attention` only support GPU version so far. As for the CPU version, we will contact PaddlePaddle team about this issue.

**Comment by sokrypton** [comment]
Any way to speed up this command?
```bash
pip install paddlepaddle-gpu==0.0.0.post112 -f https://www.paddlepaddle.org.cn/whl/linux/gpu/develop.html
```

I'm interested in developing a Google Colab notebook for helixfold-single and it appears a dev version installation is required for `fused_gate_attention`.

Unfortunately, it took over 30 minutes... :(

**Comment by LihangLiu** [comment]
> Any way to speed up this command?
> 
> ```shell
> pip install paddlepaddle-gpu==0.0.0.post112 -f https://www.paddlepaddle.org.cn/whl/linux/gpu/develop.html
> ```
> 
> I'm interested in developing a Google Colab notebook for helixfold-single and it appears a dev version installation is required for `fused_gate_attention`.
> 
> Unfortunately, it took over 30 minutes... :(

Sorry, this seems to be an issue of PaddlePaddle Installation page... An alternative is to directly visit [this link](https://www.paddlepaddle.org.cn/whl/linux/gpu/develop.html) and download the xxx.whl file  to the Colab (choose the correct xxx.whl according to your environment), and then use the following command to install:
```bash
pip install xxx.whl
```

**Comment by milkboylyf** [comment]
> Yes, `fused_gate_attention` only support GPU version so far. As for the CPU version, we will contact PaddlePaddle team about this issue.

这个问题解决了吗？现在cpu版本能使用吗fused_gate_attention这个函数吗？

**Comment by LihangLiu** [comment]
> > Yes, `fused_gate_attention` only support GPU version so far. As for the CPU version, we will contact PaddlePaddle team about this issue.
> 
> 这个问题解决了吗？现在cpu版本能使用吗fused_gate_attention这个函数吗？

你好，框架方面还没给出排期，我们也在跟进中，感谢您持续关注。
